src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -193,8 +194,9 @@ protected ElementId InternalElementId
         public virtual void Dispose()
         {
 
-            // Do not cleanup Revit elements if we are shutting down Dynamo.
-            if (DisposeLogic.IsShuttingDown)
+            // Do not cleanup Revit elements if we are shutting down Dynamo or
+            // closing homeworkspace.
+            if (DisposeLogic.IsShuttingDown || DisposeLogic.IsClosingHomeworkspace)
```

Comment:
Will that unconnect from element binder? When open the previous document, we still need to connect elements to node.